src/components/atoms/__tests__/AddButton.test.tsx

```diff
@@ -0,0 +1,13 @@
+import { getByText, render } from '@testing-library/react';
+import React from 'react';
+import AddButton from '../AddButton';
+
+test('AddButton loads with text: \'+\' and given \'className\'', () => {
+  const testClassName = 'testClassName';
+  const { container } = render((
+    <AddButton className={'testClassName'} />
```

Comment:
This will be changed to `<AddButton className='testClassName'>+</AddButton>`. I made it generic, so that you can have any content in it that you want. This is nice for when you for example want a minus button. You could then argue that it is rather strange to call it an add button, when it is in fact a minus button, but _so be it_.